README.md

```diff
@@ -12,15 +12,16 @@ Here is video of a race (running code from students):
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
 In this game, two race cars compete to achieve the most points.
-The race car have to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles, and the bonus areas.
+They need to calculate the best path to take to avoid the pitfalls.
+The participants will also have opportunities to collect bonus points.
```

Comment:
I don't think this change is better then the original text. Can you explain what was wrong with the original text?

Reply:
Its separated away from using the semicolon which is awkward with the and coming after it. just my opinion, no worries